esp/esp/themes/theme_data/bigpicture/less/main.less

```diff
@@ -274,3 +274,6 @@ div.nav-collapse.collapse {
   margin: 0;
   padding: 0;
 }
+select#regtypes {
+    font-size: 13px;
+}
```

Comment:
```suggestion
#reg_types_div {
    font-size: 13px;
}
```

Reply:
Looks like the text that is supposed to be above the <select> is also hidden, so this should catch both